spec/controllers/events_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
```

Comment:
can delete this file I think - I don't believe that controller specs are giving us any value